src/main/deltachat/index.js

```diff
@@ -62,16 +66,45 @@ class DeltaChatController extends EventEmitter {
     logCoreEv.debug(event, data1, data2)
   }
 
+  __resolveNestedMethod (self, methodName) {
+    let previous = self
+    let scope
+    for (const key of methodName.split('.')) {
```

Comment:
How about making this simpler and only allowing a 2 level nesting? We should never have more than namespace.method

Reply:
I also thought about this, I can do that.

Reply:
```
__resolveNestedMethod (self, methodName) {
    const parts = methodName.split('.')
    if (parts.length > 2) {
      const message = 'Resolving of nested method name failed: Too many parts, only two allowed: ' + methodName
      log.error(message)
      throw new Error(message)
    }
    const scope = self[parts[0]]
    if (typeof scope === 'undefined') {
      const message = 'Resolving of nested method name failed: ' + methodName
      log.error(message)
      throw new Error(message)
    }
    const method = scope[parts[1]]
    if (typeof method !== 'function') {
      const message = '(nested) Method is not of type function: ' + methodName
      log.error(message)
      throw new Error(message)
    }
    return method.bind(scope)
  }
```
Doesn't look so much simpler, does it?